documentation/sphinx/source/special-keys.rst

```diff
@@ -264,6 +263,24 @@ clients can connect FoundationDB transactions to outside events.
 #. ``\xff\xff/tracing/token := <tracing_enabled>`` Read/write. Set to true/false to enable or disable tracing for the transaction, respectively. If read, returns a 64-bit integer set to 0 if tracing has been disabled, or a random 64-bit integer otherwise (this integers value has no meaning to the client other than to determine whether the transaction will be traced).
 
+.. _special-key-space-deprecation:
+
+Deprecation
+=================
+
+Below are the deprecated special keys and related informations.
+
+#. ``\xff\xff/management/profiling/<client_txn_sample_rate|client_txn_size_limit>`` Deprecated as of 7.2. The corresponding functionalities are now covered by the global configuration module. For details, see :doc:`global-configuration`. Read/write. Changing these two keys will change the corresponding system keys ``\xff\x02/fdbClientInfo/<client_txn_sample_rate|client_txn_size_limit>``, respectively. The value of ``\xff\xff/management/client_txn_sample_rate`` is a literal text of ``double``, and the value of ``\xff\xff/management/client_txn_size_limit`` is a literal text of ``int64_t``. A special value ``default`` can be set to or read from these two keys, representing the client profiling is disabled. In addition, ``clear`` in this range is not allowed. For more details, see help text of ``fdbcli`` command ``profile client``.
+
+Versioning
+=================
+
+For how FDB clients deal with versioning, see :ref:`api-versions`. The special key space deals with the versioning by using the ``API_VERSION`` that passed to initilialize the client. The framework will ingore all modules added at the version larger than the api version set by the client. For example, if a module is added in version 7.0 but the client set the api version as 6.3, then the module it not available to the client. According to the definition of this contract, anytime you want to remove or update existing modules, you need to keep the old implementation to let the client get the same behavior when it's using the old api version.
+
+- To remove a certain module, specify the API version where the function is being deprecated in the ``registerSpecialKeySpaceModule`` function. When a client specifies an API version greater than or equal to the deprecation version, the module will not be available.
+- To update the implementation of any modules, add the new implementation and use ``API_VERSION`` to switch between different implementations.
+- Add a note in ``api-version-upgrade-guide.rst`` after you make the change.
```

Comment:
I suppose we should add a note for the module we're deprecating here.

Reply:
Added. Thanks.